### ISSUE TITLE
Removing unused import line leaves a blank line

### ISSUE THREAD

**Issue by quezak** [sweepline/eslint-plugin-unused-imports#7]
Using the `unused-imports/no-unused-imports-ts` rule, plugin version 0.1.2. Please let me know if I need to provide full config or some other data.

Looks to happen regardless of where the removed import line is positioned.

Example 1:
```typescript
// file:
import { Type } from 'class-transformer';
import { IsInt } from 'class-validator';

IsInt;

// is fixed as (note the first line is blank, should be deleted):

import { IsInt } from 'class-validator';

IsInt;
```

Example 2:
```typescript
// file:
import { Type } from 'class-transformer';
import { IsInt } from 'class-validator';
import { DeepPartial } from 'ts-essentials';

Type;
const t: DeepPartial<any>;

// is fixed as:
import { Type } from 'class-transformer';

import { DeepPartial } from 'ts-essentials';

Type;
const t: DeepPartial<any>;
```

Example 3:
```typescript
// file:
import { Type } from 'class-transformer';
import { IsInt } from 'class-validator';

Type;

// Is fixed as (note double blank line, I've temporarily disabled other rules that would fix this case)
import { Type } from 'class-transformer';


Type;
```

**Comment by quezak** [comment]
Note: this problem also applies to removing one item from a multiline import:
``` typescript
// file:
import {
    Dictionary,
    OmitProperties,
    SafeDictionary,
    ValueOf,
} from 'ts-essentials';

const x: ValueOf<SafeDictionary<OmitProperties<{}, any>>>;

// is fixed as:
import {

    OmitProperties,
    SafeDictionary,
    ValueOf,
} from 'ts-essentials';

const x: ValueOf<SafeDictionary<OmitProperties<{}, any>>>;
```

**Comment by IAMtheIAM** [comment]
It would be nice if this cleaned the line-break but not essential.

however based on your tip [here](https://github.com/sweepline/eslint-plugin-unused-imports/issues/6#issuecomment-607347333) using `no-multi-spaces` and `no-multi-empty-lines` set at `1` works perfectly for removing the spaces and new lines!

Thanks for the cool plugin!
